core/src/test/java/kafka/log/remote/RemoteLogManagerTest.java

```diff
@@ -220,8 +220,8 @@ void testRemoteLogMetadataManagerWithEndpointConfig() {
 
         ArgumentCaptor<Map<String, Object>> capture = ArgumentCaptor.forClass(Map.class);
         verify(remoteLogMetadataManager, times(1)).configure(capture.capture());
-        assertEquals(host + ":" + port, capture.getValue().get("bootstrap.servers"));
-        assertEquals(securityProtocol, capture.getValue().get("security.protocol"));
+        assertEquals(host + ":" + port, capture.getValue().get("remote.log.metadata.common.client.bootstrap.servers"));
+        assertEquals(securityProtocol, capture.getValue().get("remote.log.metadata.common.client.security.protocol"));
```

Comment:
Any reason why don't we use `REMOTE_LOG_METADATA_COMMON_CLIENT_PREFIX` like other places did?

Reply:
Sure, I was following the text content instead, but we can use the constant here as well. Applying fix